Gemfile

```diff
@@ -8,6 +8,8 @@ end
 gem 'rails', '~> 5.1'
 
 gem 'autoprefixer-rails'
+gem 'axlsx', '2.1.0.pre'
```

Comment:
I simply copy&pasted from their github readme. But It seems to run with no version requirement at all here.